analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DeadStores.CSharp11.cs

```diff
@@ -114,9 +114,20 @@ void NewlinesInStringInterpolation(string param)
 
     string empty = "";
     string x = $"{empty +
-        empty}"; // Noncompliant@-1 FP (string is still empty, should be compliant)
+        empty}"; // Noncompliant@-1 string interpolation values are intentionally not evaluated
     x = "Test";
     Foo(x);
 }
 
+void IgnoredValues()
+{
+    string emptyMultilineRawStringLiteral = $$"""
+
+        """; // Compliant
+    emptyMultilineRawStringLiteral = "other";
+
+    // Variables should be used in order the rule to trigger
+    Console.WriteLine("", emptyMultilineRawStringLiteral);
```

Comment:
why not use `Foo`, and avoid the comment? (the comment applies to the whole file)